features/languages-testcase.feature

```diff
@@ -0,0 +1,39 @@
+         Feature: Languages
+
+Scenario Outline: User is able to change the language
+           Given User sees <country-flag> on the pah-fm website
+            When User clicks on <country-flag>
+            Then The language on the page changes to <chosen-language>
+
+        Examples:
+|    country-flag | chosen-language |
+|         Ukraine |       Ukrainian |
+|  United Kingdom |         English |
+|          Poland |          Polish |
+
+Scenario Outline: Language is retained after User login
+           Given User is on the login page
+             And User chooses <chosen-language> by clicking on a flag
+            When User log in <status>
+            Then <outcome-language> is the same as <chosen-language>
+
+         Examples:
```

Comment:
Whould we like to have step with unsuccessful login? What is the difference after clicking login (and failing) than just being on login website?

Reply:
My point was to create a testcase preventing a random change of the language after an unsuccessful login (e.g. I put incorrect password and suddenly I see an error message and everything else in Ukrainian). Should I remove it?

Reply:
You can always add some comments or user stories for reference (it would be easier for other people to read & understand the features).

Reply:
Not sure how user stories should be created yet - I will try to create one, but it might be messy

Reply:
Please do. Easiest template is:

As a <role> 
I can <capability>, 
so that <receive benefit>